eng/scripts/dependencies/compare-deps-files.ps1

```diff
@@ -0,0 +1,53 @@
+param(
+  [string]$PSDepsFile,
+  [string]$AzSdkDepsFile
+)
+
+function Get-DepsFileContent($depsfile)
+{
+  if ($depsfile.StartsWith("http")) {
+    try {
+      $response = Invoke-WebRequest -Uri $depsfile
+      $content = $response.Content
+    }
+    catch {
+      $statusCode = $_.Exception.Response.StatusCode.value__
+      Write-Error "Failed to download deps file [$statusCode] $depsfile"
```

Comment:
This won't terminate the script unless the `ErrorPreference` is "Stop". Did you want it to terminate? Use `throw "string message"`.